org.adempiere.base/src/org/compiere/model/MTable.java

```diff
@@ -601,6 +601,21 @@ public PO getPO (ResultSet rs, String trxName)
 		return po;
 	}	//	getPO
 
+	/**
+	 * Get PO Class Instance
+	 * 
+	 * @param  uuID    UUID
+	 * @param  trxName transaction
+	 * @return         PO for Record or null

```

Comment:
this javadoc is wrong, it will never return null

Reply:
@hengsin  you may referencing old commit as I did remove "or null" in commit 408fe45 (Latest one)